backend/game/engine/board.py

```diff
@@ -16,28 +16,26 @@ def start_game(self):
         self.players.set_player_order()
         self.STATUS = status.ACTIVE
 
-    def check_if_winner(self, player):
-        if player.victory_points == constants.VICTORY_POINTS_TO_WIN or all(
-                alive_player == player for alive_player
-                in self.players.get_alive_player()):
-            self.WINNER = player
+    def check_if_winner(self, potential_winner):
+        if (potential_winner.victory_points == constants.VICTORY_POINTS_TO_WIN or
+                players.if_last_player_alive(potential_winner)):
+            self.WINNER = potential_winner
             self.end_game()
 
     def end_game(self):
         self.STATUS = status.COMPLETED
 
-    def turn(self, current_player):
-        # roll dice & dice_resoultion method
-        pass
+    def turn(self, active_player):
+        if active_player == self.players.get_current_player():
```

Comment:
This isn't pythonic, the convention would be if not blah blah blah, throw exception. Forget the else case, just unindent the code and carry on.

Reply:
Sure but I do want to be careful with how we handle these exceptions before we go down the dangerous rabbit hole of creating custom exceptions for everything.